Makefile

```diff
@@ -397,10 +397,24 @@ docker-test: docker-build	## execute a smoke test of the built Docker image (val
 	curl localhost:$(APP_PORT) | grep "$(APP_NAME)"
 	docker-compose $(DOCKER_TEST_COMPOSES) stop
 
+DOCKER_COMPOSES := -f "$(APP_ROOT)/docker/docker-compose.yml" -f "$(APP_ROOT)/docker/docker-compose.override.yml"
+.PHONY: docker-up
+docker-up: docker-build   ## run all containers using compose
+	docker-compose $(DOCKER_COMPOSES) up
+
+DOCKER_DEV_COMPOSES := -f "$(APP_ROOT)/docker/docker-compose.yml" -f "$(APP_ROOT)/docker/docker-compose.dev.yml"
```

Comment:
Yeah I thought about making something fancy that would create the .yml from the example if they don't exist, but since we have the same "issue" with the cowbird.ini being used while still comitting only the .example, I guess that's the way to go. A dev need either to create it's own file or copy the example ones.

Reply:
While I'm not sure it's really necessary, we could make a target, say "docker-dev-setup", that looks for the files, and creates them from the `.example` files if not found.

Reply:
I think that the `. yml.examples` should be used directly. 
Best way to make sure the examples are valid is by using them in real use cases.